res/css/views/settings/tabs/user/_LabsUserSettingsTab.scss

```diff
@@ -19,7 +19,8 @@ limitations under the License.
         margin-top: 32px;
 
         .mx_SettingsFlag {
-            margin-right: 0; // remove right margin to align with beta cards
+            margin-right: 0; // remove right margin to al
+            padding-top: 2.5px;
```

Comment:
fractional pixels are never fun, and this will fall out of alignment with the font size changing (Settings > Appearance) - maybe look into vertical-alignment, flexboxes, align-items, etc

Reply:
> 
Sure, I'll look into it!

Reply:
> fractional pixels are never fun, and this will fall out of alignment with the font size changing (Settings > Appearance) - maybe look into vertical-alignment, flexboxes, align-items, etc

@t3chguy can you verify the changes?, this time I have used flexbox and alignment :)